### DISCUSSION TITLE
K8s cannot sync entire registry

### DISCUSSION THREAD

**Discussion by mdolny** [xelalexv/dregsy#40] [Q&A]
Hi, 
How can I deploy dregsy to my k8s cluster as complete registry sync?
Something like that:
    - from: *
 `  
`level=error msg="error parsing config file '/config/config.yaml': yaml: line 10: did not find expected alphabetic or numeric character"`

**Comment by xelalexv** [comment]
Hi,
matching images is an upcoming feature, see issue #16. You can find an alpha implementation in branch `filter`, and an accompanying Docker image tagged `filter`. See [usage instructions here](https://github.com/xelalexv/dregsy/blob/filter/doc/design-image-matching.md).
